src/flint.h.in

```diff
@@ -31,6 +31,7 @@
 #else
 # include "flint-config.h"
 #endif
+#include "flint-mparam.h"
```

Comment:
I don't think flint-mparam.h should be included in flint.h because it could be quite large in the future.